test/icu.macro.spec.js

```diff
@@ -314,80 +314,80 @@ pluginTester({
       );
     `,
     `
-      import { Trans } from "../icu.macro";
+      import { Trans } from "../../../icu.macro";
 
       const x = <Trans>Welcome, &quot;{ name }&quot;!</Trans>
     `,
     {
       code: `
         import React from "react"
-        import { Trans, number } from "../icu.macro";
+        import { Trans, number } from "../../../icu.macro";
 
         const count = 2;
         const outside = number\`\${count}\`;
       `,
       snapshot: false,
-      error: /"number``" can only be used inside <Trans> in "[^"]+" on line 5/,
+      error: /"number``" can only be used inside <Trans> in "[^"]+" on line 6/,
```

Comment:
I'm not entirely sure about this one but I think babel-plugin-tester v11 either changed now it reports errors or adds an line to the top of files it tests because all the errors are now reported on the line below what they did in previous versions.